src/main/java/mate/academy/springboot/datajpa/dto/ProductResponseDto.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.springboot.datajpa.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class ProductResponseDto {
+    private Long id;
+    private String title;
+    private BigDecimal price;
+    private CategoryResponseDto category;
```

Comment:
```suggestion
    private Long categoryId;
```

Reply:
let's not include all info about category
if client will want this info, it will send additional request with id of category